packages/profile/src/components/Dialog/DialogMsg.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
```

Comment:
@vaibhavsTekdi We already have dialog for showing the "Attendance" submitted successfully right? Can not we use that?

Reply:
@vaivk369 This dialog is included in profile module as the attendance gets marked in that module. The other dialog was included in cohort to show message only.